### PR TITLE
Update LinearGradient docs

### LINKED ISSUE #18484: LinearGradient documentation: Possible to add an image of what the end results look like?
https://docs.flutter.io/flutter/painting/LinearGradient-class.html

In g3, we had a client building a prototype who was unsure of how to get a gradient to cut off before the full height of the screen.  They used `stops` to try and achieve this effect instead of the `begin` and `end` Alignment parameters.

I suspect that having an image in the `LinearGradient` documentation to show what the gradient looks like.

Optionally, I think it would be valuable to include a sample that uses the default `tileMode` since that will make it easier to illustrate how the begin and end properties work.

### PR DESCRIPTION
**Pull request by haeseoklee**
## Description

To make the result easier to understand, I replaced the existing snippet with dart pad snippet. 

And I also changed the gradient color in the snippet code.

## Related Issues

Fixes https://github.com/flutter/flutter/issues/18484

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/painting/gradient.dart [hunk 1]
```diff
@@ -331,22 +331,23 @@ abstract class Gradient {
...
-///     ),
-///   ),
-/// )
+///  Widget build(BuildContext context) {
+///    return Container(
+///      decoration: BoxDecoration(
+///        gradient: LinearGradient(
+///          begin: Alignment.topLeft,
+///          end: Alignment(0.8, 0.0), // 10% of the width, so there are ten blinds.
+///          colors: [const Color(0xffee0000), const Color(0xffeeee00)],
```
Comment (goderbauer) [inline packages/flutter/lib/src/painting/gradient.dart:345]: Can you add back the comment describing the colors? Thanks.

**Review by goderbauer — APPROVED** [review]
LGTM after comment is resolved.

File: packages/flutter/lib/src/painting/gradient.dart (hunk 1 above)
Comment (haeseoklee) [reply packages/flutter/lib/src/painting/gradient.dart:345]: @goderbauer 
Thank you for your information.

**Comment by haeseoklee** [comment]
@goderbauer 
I added a comment. Thank you for checking.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**